draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1558,20 +1558,12 @@ A comprehensive list of known implementations and integrations can be found at [
 
 # Security Considerations
 
-AEGIS-256 offers 256-bit message security against plaintext and state recovery, whereas AEGIS-128L offers 128-bit security.
-
-An authentication tag may verify under multiple keys, nonces, or associated data, but AEGIS is assumed to be key committing in the receiver-binding game, preventing common attacks when used with low-entropy keys such as passwords. Finding distinct keys and/or nonces that successfully verify the same `(ad, ct, tag)` tuple is expected to require ~2<sup>64</sup> attempts with a 128-bit authentication tag and ~2<sup>128</sup> attempts with a 256-bit tag.
+## Usage Guidelines
 
-It is fully committing in the restricted setting where an adversary cannot control the associated data. As shown in {{IR23}}, with the ability to alter the associated data, it is possible to efficiently find multiple keys that will verify the same authenticated ciphertext.
-
-Protocols mandating a fully committing scheme without that restriction can provide the associated data as input to a cryptographic hash function and use the output as the `ad` parameter of the `Encrypt` and `Decrypt` functions. The selected hash function must ensure a minimum of 128-bit preimage resistance. An instance of such a function is SHA-256 {{!RFC6234}}.
-
-Under the assumption that the secret key is unknown to the attacker, all AEGIS variants target 128-bit security against forgery attacks regardless of the tag size.
+### Key And Nonce Selection
```

Comment:
I do think the `And` should be `and` based on [RFC 7322](https://datatracker.ietf.org/doc/html/rfc7322#section-3.4):

> Typically, all words in a title will be capitalized, except for internal articles, prepositions, and **conjunctions**.

Reply:
I wasn't sure about that. Thanks for the reference!

`And` has been changed to `and`.